### ISSUE TITLE
Deprecated DT call

### ISSUE THREAD

**Issue by Giannettig** [jayjacobs/verisr#24]
I tried to run the RMD file and consistently I have been rewarded with this error: 

> with=FALSE together with := was deprecated in v1.9.4 released Oct 2014. Please wrap the LHS of := with parentheses; e.g., DT[,(myVar):=sum(b),by=a] to assign to column name(s) held in variable myVar. See ?':=' for other examples. As warned in 2014, this is now a warning.with=FALSE together with := was deprecated in v1.9.4 released Oct 2014. Please wrap the LHS of := with parentheses; e.g., DT[,(myVar):=sum(b),by=a] to assign to column name(s) held in variable myVar. See ?':=' for other examples. As warned in 2014, this is now a warning.with=FALSE together with := was deprecated in v1.9.4 released Oct 2014. Please wrap the LHS of := with parentheses; e.g., DT[,(myVar):=sum(b),by=a] to assign to column name(s) held in variable myVar. See ?':=' for other examples. As warned in 2014, this is now a warning.with=FALSE together with := was deprecated in v1.9.4 released Oct 2014. Please wrap the LHS of := with parentheses; e.g., DT[,(myVar):=sum(b),by=a] to assign to column name(s) held in variable myVar. See ?':=' for other examples. As warned in 2014, this is now a warning.

I believe the library uses some deprecated DT call on some function.

**Comment by onlyphantom** [comment]
I don't believe it's maintained anymore - but I rewrote a lot of the functions in pure base R (the underlying code for this package is `data.table`) and only where necessary, `dplyr`. All the core functionalities plus some are supported and compatible with later versions of R, tested on R 3.5+. 

You can install the package from here: https://github.com/onlyphantom/verisr2 and feel free to open any issues!
